notebooks/21 Unsupervised learning - Hierarchical and density-based clustering algorithms.ipynb

```diff
@@ -2,27 +2,14 @@
  "cells": [
   {
    "cell_type": "code",
-   "execution_count": 1,
+   "execution_count": null,
    "metadata": {
-    "collapsed": false
-   },
-   "outputs": [
-    {
-     "name": "stdout",
-     "output_type": "stream",
-     "text": [
-      "Andreas Mueller, Sebastian Raschka \n",
-      "last updated: 2016-07-08 \n",
-      "\n",
-      "CPython 3.5.1\n",
-      "IPython 4.2.0\n",
-      "\n",
-      "numpy 1.11.0\n",
-      "scipy 0.17.1\n",
-      "matplotlib 1.5.1\n"
-     ]
+    "collapsed": false,
+    "nbpresent": {
```

Comment:
It does nothing. I believe I was checking out the nbpresent functionality but didn't actually use it. Still, the ipynb added this metadata. Not a big deal, I don't think.